src/qoi/src/qoi_factory.C

```diff
@@ -99,6 +99,9 @@ namespace GRINS
   {
     QoIBase* qoi = NULL;
 
+    // TODO better way to do this? (skip the final add_qoi at the end of the if-else)
```

Comment:
@pbauman or anyone else, is there a better way to do this?

I have to add the qois to the composite_qoi in a loop at line 243, but there is also a final add at the end of the function (line 258). So if I don't skip that for this case, I would be duplicating the last qoi.

Reply:
Yeah, I saw this. The way to actually fix this is make this a proper factory pattern and encapsulate the QoI builds better. But that's beyond the scope of this PR. I can live with this hack for now.

Reply:
okay, I'll remove the TODO and then we should be set